models/dalec/R/met2model.DALEC.R

```diff
@@ -96,7 +96,8 @@ met2model.DALEC <- function(in.path, in.prefix, outfolder, start_date, end_date,
     ifelse(leap_year(year)==TRUE,
            dt <- (366*24*60*60)/length(sec), #leap year
            dt <- (365*24*60*60)/length(sec)) #non-leap year
-    tstep = 86400/dt
+    tstep = round(86400/dt)
+    dt = 86400/tstep
```

Comment:
Same comment as above. Might be worth setting some of these common constants as package variables, as per [this Stack Overflow answer](http://stackoverflow.com/a/12605694).